osquery/tables/system/windows/chocolatey_packages.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <boost/filesystem.hpp>
+#include <boost/property_tree/xml_parser.hpp>
+
+#include <osquery/core.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/core/process.h"
```

Comment:
Why do you need these two (process, conversions) includes?

Reply:
Ah, `process.h` was for `getEnvVar`. I was using `osquery::split` earlier but found it wasn't needed, and forgot to remove the header. Thanks!